sync/http/component.go

```diff
@@ -121,3 +122,149 @@ func (c *Component) createHTTPServer() *http.Server {
 		Handler:      routerAfterMiddleware,
 	}
 }
+
+// propSetMsg is used to log successes when using
+// the HTTP Builder setters, in a uniform way
+const propSetMsg = "Setting property '%v' for '%v'"
+
+// Builder gathers all required and optional properties, in order
+// to construct an HTTP component
```

Comment:
All comments should end with a punctuation.
https://github.com/golang/go/wiki/CodeReviewComments

Reply:
Done.